From 4a7a8a8a6d3dc9466eb437517d16e8ab0ac52704 Mon Sep 17 00:00:00 2001
From: Germano Percossi <germano.percossi@citrix.com>
Date: Fri, 18 Nov 2016 18:54:52 +0000
Subject: [PATCH 11/12] mount.cifs: Remove unneeded stdbool header include

Signed-off-by: Germano Percossi <germano.percossi@citrix.com>
(cherry picked from commit 743301da5bf502197beb1f91f98e11eea9070e4e)

Resolves bz: 1427337

Signed-off-by: Sachin Prabhu <sprabhu@redhat.com>
---
 mount.cifs.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/mount.cifs.c b/mount.cifs.c
index 6eb0e6b..b2bcb7f 100644
--- a/mount.cifs.c
+++ b/mount.cifs.c
@@ -45,7 +45,6 @@
 #include <libgen.h>
 #include <sys/mman.h>
 #include <sys/wait.h>
-#include <stdbool.h>
 #ifdef HAVE_SYS_FSUID_H
 #include <sys/fsuid.h>
 #endif /* HAVE_SYS_FSUID_H */
-- 
2.9.3

